docs/en/observability/cloud-monitoring/aws/monitor-amazon-kinesis.asciidoc

```diff
@@ -0,0 +1,257 @@
+[[monitor-amazon-kinesis]]
+= Monitor Amazon Kinesis data streams
+
+++++
+<titleabbrev>Monitor Kinesis data streams</titleabbrev>
+++++
+
+Amazon Kinesis Data Streams is a serverless streaming data service that can
```

Comment:
Add generic link to info about the service. Make sure the link is "future proof" (unlikely to change)